frontend/public/components/monitoring.tsx

```diff
@@ -1064,13 +1065,49 @@ const PollerPages = () => {
   </Switch>;
 };
 
-export const MonitoringUI = () => <Switch>
+export const MonitoringUI = ({match}) => <Switch>
   <Redirect from="/monitoring" exact to="/monitoring/alerts" />
   <Route path="/monitoring/query-browser" exact component={QueryBrowserPage} />
   <Route path="/monitoring/silences/~new" exact component={CreateSilence} />
+  <Route path="/monitoring/(alertmanageryaml|alerts|silences)" exact render={() => <AlertingPage match={match} />} />
   <Route component={PollerPages} />
 </Switch>;
 
+const AlertingPage: React.SFC<AlertingPageProps> = ({match}) => {
+  const AlertPath = '/monitoring/alerts';
+  const SilencePath = '/monitoring/silences';
+  const YAMLPath = '/monitoring/alertmanageryaml';
+  return <React.Fragment>
+    <div className="co-m-nav-title co-m-nav-title--detail">
+      <h1 className="co-m-pane__heading">
+        <div className="co-m-pane__name co-resource-item">
+          <span id="resource-title" className="co-resource-item__resource-name">Alerting</span>
+          <HeaderAlertmanagerLink path="/#/alerts" />
+        </div>
+      </h1>
+    </div>
+    <div className="co-m-horizontal-nav__menu">
+      <ul className="co-m-horizontal-nav__menu-primary">
+        <li
+          className={classNames('co-m-horizontal-nav__menu-item', {'co-m-horizontal-nav-item--active': match.path === AlertPath})}>
+          <Link to={AlertPath}>Alerts</Link>
+        </li>
+        <li
+          className={classNames('co-m-horizontal-nav__menu-item', {'co-m-horizontal-nav-item--active': match.path === SilencePath})}>
+          <Link to={SilencePath}>Silences</Link>
+        </li>
+        <li
+          className={classNames('co-m-horizontal-nav__menu-item', {'co-m-horizontal-nav-item--active': match.path === YAMLPath})}>
+          <Link to={YAMLPath}>YAML</Link>
+        </li>
+        <li className="co-m-horizontal-nav__menu-item co-m-horizontal-nav__menu-item--divider"></li>
+      </ul>
+    </div>
+    {match.path === YAMLPath && <Route component={AlertManagerYAML} />}
```

Comment:
I don't think the `<Route>`s are doing any useful here.

Reply:
thanks, yeah, that didn't look right, switched to `<Switch/>`